package.json

```diff
@@ -2,21 +2,19 @@
   "name": "tic-tac-toe-frontend",
   "version": "0.1.0",
   "private": true,
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject",
-    "prettier": "prettier --write ."
-  },
+  "proxy": "http://localhost:8086",
```

Comment:
What does this do?

Reply:
We need it when we use fetch.

Reply:
There are downsides to hardcoding the url in the following manner. But we can leave it like this for now.